holoviews/plotting/bokeh/element.py

```diff
@@ -583,21 +587,23 @@ def current_handles(self):
             handles.append(plot.title)
 
         if self.current_frame:
-            if self.subplots:
-                current_frames = [(sp.current_frame if isinstance(sp.current_frame, Element)
-                                   else sp.current_frame.values()[0])
-                                  for sp in self.subplots.values()
-                                  if sp.current_frame]
-                framewise = any(self.lookup_options(frame, 'norm').options.get('framewise')
-                                for frame in current_frames)
-            else:
-                opts = self.lookup_options(self.current_frame, 'norm')
-                framewise = opts.options.get('framewise')
-            if framewise or isinstance(self.hmap, DynamicMap):
+            if self.framewise or isinstance(self.hmap, DynamicMap):
                 handles += [plot.x_range, plot.y_range]
         return handles
 
 
+    @property
+    def framewise(self):
+        """
+        Property to determine whether the current frame should have
+        framewise normalization enabled.
+        """
```

Comment:
Docstring should say why this is a bokeh specific property - that it is about generating events to update the bokeh plots when it makes sense. Otherwise, having this property makes sense.